zproject/settings.py

```diff
@@ -178,6 +178,7 @@ def get_secret(key):
                     'SYSTEM_ONLY_REALMS': {"zulip.com"},
                     'FIRST_TIME_TOS_TEMPLATE': None,
                     'USING_PGROONGA': False,
+                    'INLINE_LINKS_EMBED': True,
```

Comment:
This should sit next to `INLINE_IMAGE_PREVIEW`, and probably be called `settings.INLINE_URL_EMBED_PREVIEW`.